taskcluster/ci/ui-test-apk/kind.yml

```diff
@@ -270,10 +271,11 @@ tasks:
                 GOOGLE_PROJECT: moz-fenix
     fenix-arm-nightly:
         attributes:
-            build-type: nightly-firebase
+            build-type: fenix-nightly-firebase
             shipping-product: fenix
         description: Test Fenix
         run-on-tasks-for: [github-push]
+        run-on-git-branches: [main]
```

Comment:
Previously, this was running everywhere including `releases` branch (eg, here https://github.com/mozilla-mobile/firefox-android/runs/12432236031).